### PR TITLE
feat: add Checkbox component to auth page form and designed the auth layout

### PR DESCRIPTION
**Pull request by sidarth-23**
The Checkbox component has been added to the auth page form to align with the design of the page. It allows users to select options with a checkbox.

## Summary

- [ ] Tested (Must)
- [ ] Test Case added
- [ ] Build Successful (Must)
- [ ] Sufficient Code comments added (Must)
- [ ] Attached Screenshots / Videos 
- [ ] All Relevant Documents added

### CODE REVIEW

File: packages/ui/src/partials/checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+"use client"
```
Comment (yogai53) [inline packages/ui/src/partials/checkbox.tsx:1]: New PR please.

File: packages/ui/package.json [hunk 2]
```diff
@@ -32,10 +32,12 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
+    "@radix-ui/react-checkbox": "^1.0.4",
```
Comment (yogai53) [inline packages/ui/package.json:35]: new PR

File: packages/ui/src/components/auth-page/auth-page.tsx [hunk 3]
```diff
@@ -57,8 +62,25 @@ function PageWrapper({
  *
  * @param children - The content of the input fields
  */
-function InputsWrapper({ children }: { children: React.ReactNode }) {
-  return <ul className="w-full max-w-md">{children}</ul>;
+function FormWrapper({
```
Comment (yogai53) [inline packages/ui/src/components/auth-page/auth-page.tsx:65]: Can it be a new component

Comment (yogai53) [reply packages/ui/src/components/auth-page/auth-page.tsx:65]: New file.

File: packages/ui/src/components/auth-page/auth-page.tsx [hunk 4]
```diff
@@ -78,17 +100,52 @@ function InputField({
...
 
+/**
+ * This is checkbox field which is aligned with the design of the auth page
+ *
+ * @param fieldName - The name of the field
+ * @param type - The type of the input field
+ * @param placeholder - The placeholder of the input field (optional)
+ */
+
+function CheckboxField({
```
Comment (yogai53) [inline packages/ui/src/components/auth-page/auth-page.tsx:122]:
```suggestion
function RememberMeCheckboxField({
```

File: packages/ui/src/components/auth-page/auth-page.tsx [hunk 5]
```diff
@@ -119,33 +192,94 @@ function ButtonElement({ contentIn }: { contentIn: string }) {
...
+}: {
+  provider: "github" | "google";
+}) {
+  return (
+    <Button
+      variant="outline"
+      className="grow bg-white text-black hover:bg-black/20 flex gap-3"
+    >
+      {provider === "github" && (
+        <svg
```
Comment (yogai53) [inline packages/ui/src/components/auth-page/auth-page.tsx:210]: Create separate icon file.

File: packages/ui/src/components/auth-page/auth-page.tsx
Comment (yogai53) [inline packages/ui/src/components/auth-page/auth-page.tsx:1]: See if some of the components here can be moved to a new file or folder, if they are more generate and not auth specific.

**Review by yogai53 — CHANGES_REQUESTED** [review]
Please seggregate this into two PRs.

**Comment by sidarth-23** [comment]
Closing this as we changed the course of action of development

**Pull request closed without merging**